css/products.css

```diff
@@ -0,0 +1,200 @@
+/* single-column layout */
+body {
```

Comment:
it is better to use wrapper rather than body element to style layout

Reply:
Wrapper added to body element :+1:

Reply:
Wrapper added to body element :+1: